### ISSUE TITLE
Inconsistency Graph/Disasm

### ISSUE THREAD

**Issue by Maijin** [hteso/iaito#90]
.string

![image](https://cloud.githubusercontent.com/assets/1408600/24706269/6679bc88-1a0f-11e7-89a5-a73aa8ed014f.png)
![image](https://cloud.githubusercontent.com/assets/1408600/24706272/68b61c4e-1a0f-11e7-85a2-79499cc020de.png)

[biglib.obj.zip](https://github.com/hteso/iaito/files/896222/biglib.obj.zip)

**Comment by mrexodia** [comment]
That string is pretty random haha

**Comment by Maijin** [comment]
It is. But that's not the point of the issue ;) This will be reproducible in other cases when user define this manually. Since it's not yet possible via the interface well... This did the trick :)

**Comment by mrexodia** [comment]
Interesting, but isn't this a bug in r2? Because that's what provides the
graph view right?

**Comment by Maijin** [comment]
nop
![image](https://cloud.githubusercontent.com/assets/1408600/24706843/88ac51e2-1a11-11e7-9d74-b0a5cb01eeb8.png)
![image](https://cloud.githubusercontent.com/assets/1408600/24706845/8ab9bae2-1a11-11e7-813a-7995d26fd861.png)

**Comment by mrexodia** [comment]
Yeah you obviously have to use the Web graph... I didn't think iaito had
custom code for the graph

**Comment by mrexodia** [comment]
Nevermind I was blind...

**Comment by Maijin** [comment]
Apparently Web Graph, Iaito graph are different so it's not even relevant to test :/

**Comment by pwntester** [comment]
can you check if the string appears in `agj` output?

**Comment by radare** [comment]
no doesnt appears because agj only shows the insturction disasm, not the string that the instruction references or the comments or so.

maybe we can extend the json to include this information
